web/src/i18n/translations/en.yml

```
@@ -1,556 +1,557 @@
-Successfully created webhook!: ''
-Failed to delete webhook.: ''
-Successfully deleted webhook!: ''
-Failed to update webhook.: ''
-Successfully updated webhook!: ''
-Failed to create integration.: ''
-Successfully created integration!: ''
-Failed to create project.: ''
-Successfully created project!: ''
-Failed to delete workspace.: ''
-Successfully deleted workspace!: ''
+Download: ''
```

yamllint

[error] wrong new line character: expected \n

(new-lines)